src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -158,7 +158,7 @@ uiModules.get('apps/management')
           return service.get().then(function (obj) {
             obj.id = doc._id;
             return obj.applyESResp(doc).then(function () {
-              return obj.save();
+              return obj.save(true);
```

Comment:
I usually try to stick to having an `options` object parameter in a function definition, and have the calling code use something like:

```
obj.save({ confirmOverwrite: true });
```

It helps readability a ton, because a simple boolean parameter is always ambiguous. There are a lot of things the reader might think it means.